inc/customizer.php

```diff
@@ -290,6 +290,24 @@ public static function register( $wp_customize ) {
 				'type'     => 'checkbox',
 			)
 		);
+		
+		// Show comment author's full name in reply-link
+		$wp_customize->add_setting(
+			'independent_publisher_general_options[show_comment_authors_full_name_in_reply_link]', array(
+				'default'           => false,
+				'type'              => 'option',
+				'capability'        => 'edit_theme_options',
+				'sanitize_callback' => 'independent_publisher_sanitize_checkbox',
+			)
+		);
+		$wp_customize->add_control(
+			'show_comment_authors_full_name_in_reply_link', array(
+				'settings' => 'independent_publisher_general_options[show_comment_authors_full_name_in_reply_link]',
+				'label'    => __( 'Show comment author\'s full name in reply-link', 'independent-publisher' ),
```

Comment:
I suggest changing the label here to "Show Full Name in Comment Reply-to":

![2016-07-16_14-24-37](https://cloud.githubusercontent.com/assets/53005/16896428/3644a96a-4b61-11e6-9baf-45d4e5a02379.png)

That will fit better in the Customizer than what you have now:

![2016-07-16_14-25-04](https://cloud.githubusercontent.com/assets/53005/16896425/3236abac-4b61-11e6-8330-c1afb10ecbbb.png)